dcicutils/secrets_utils.py

```diff
@@ -0,0 +1,55 @@
+import os
+import boto3
+import json
+from botocore.exceptions import ClientError
+from dcicutils.ecr_utils import CGAP_ECR_REGION
+from dcicutils.misc_utils import PRINT
+
+
+# Environment variable whose value is the name of a secret
+# stored in AWS Secrets Manager that contains the global
+# application configuration. This secret JSON is acquired
+# and returned to the caller. Note that this API can behave differently
+# across AWS Accounts.
+GLOBAL_APPLICATION_CONFIGURATION = 'IDENTITY'
+
+
+def assume_identity():
+    """ Grabs application identity from the secrets manager.
+        Looks for environment variable IDENTITY, which should contain the name of
+        a secret in secretsmanager that is a JSON blob of core configuration information.
+        Default value is current value in the test account. This name should be the
+        name of the environment.
+    """
+    secret_name = os.environ.get(GLOBAL_APPLICATION_CONFIGURATION, 'dev/beanstalk/cgap-dev')
+    region_name = CGAP_ECR_REGION  # us-east-1, must match ECR/ECS Region
+    session = boto3.session.Session(region_name=region_name)
+    client = session.client(
+        service_name='secretsmanager',
+        region_name=region_name
+    )
+
+    try:
+        get_secret_value_response = client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:  # leaving some useful debug info to help narrow issues
+        if e.response['Error']['Code'] in [
```

Comment:
At some point this list should perhaps become a variable, but the code is evolving so at your leisure.